chordspy/chords_serial.py

```
@@ -0,0 +1,280 @@
+            while retry_counter < self.retry_limit:   # Try to identify the board with retries
+                self.ser.write(b'WHORU\n')            # Send identification command
+                try:
+                    response = self.ser.readline().strip().decode()
+                except UnicodeDecodeError:
+                    response = None
+
+                if response in self.supported_boards:  # Board identified successfully
+                    self.board = response
+                    print(f"{response} detected at {port} with baudrate {baudrate}")
+                    self.num_channels = self.supported_boards[self.board]["Num_channels"]
+                    sampling_rate = self.supported_boards[self.board]["sampling_rate"]
```

Ruff (0.12.2)

Local variable `sampling_rate` is assigned to but never used

Remove assignment to unused variable `sampling_rate`

(F841)